mccurdy342/4fydp
# Jan14 base 1h55mins

Changes from last term:
-fix infeasibility
--change input parameters for teacher availiblity
-at yijk varaible so cohorts only get 1 teacher per subject
-remove slack and surplus for min reqs and reduce min reqs
-add constraint so primary doesnt get french
-added subjects drama and music

Changes from this week:
-cleaned up the min reqs to remove slack and surplus
-forced language for junior/senior to be at least once a day
-forced french to be less than once a day to evenly distribute
-added constraints to evenly distribute classes (gym,social studies, scinece) throughout the week using slack and surplus varibables. these are commented because still testing how long they will take. these are commented in the objective function too
-added a constraint for back-back but dont think it will work/ dont think its exactly what we want. left it commented for future use

- takes about 1h 55 mins to run on my computer, aboout 1 hour on Allisons 

To test:
-run the model and paste results into excel
-verify the above constraints are met
1. make sure no primary class is assigned french
2. for a few cohorts, filter buy subject to make sure they only have one teacher per subject
3. for a few junior cohorts, check to see if they get language once a day and french not more than once a day
